QEfficient/cross_compile.py

```diff
@@ -0,0 +1,159 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c)  2023-2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+
+import json
+import os
+import subprocess
+from typing import List, Tuple
+
+from QEfficient.utils.logging_utils import logger
+
+
+def create_and_dump_specializations(batch_size: int, prompt_len: int, ctx_len: int, path: str):
+    # Create
+    specializations = {
+        "specializations": [
+            {
+                "batch_size": str(batch_size),
+                "seq_len": str(prompt_len),
+                "ctx_len": str(ctx_len),
+            },
+            {"batch_size": str(batch_size), "seq_len": "1", "ctx_len": str(ctx_len)},
+        ]
+    }
+    # Dump
+    with open(path, "w") as file:
+        json.dump(specializations, file, indent=4)
+
+
+def compile_kv_model_on_cloud_ai_100(
```

Comment:
Why the file is named `cross_compile`, also anything special this does, could have been handled with compile only right?
better to keep the compiled qpc command creation in the utils?

Reply:
will add compile dir